css/core.scss

```diff
@@ -107,6 +107,12 @@ a.btn:hover {
   font-family: 'Source Code Pro', monospace;
 }
 
+@media (max-width: 447px){ 
+  .btn-lg { 
+    margin: 1rem;
```

Comment:
I'd prefer to remove `mr-lg-3` from the affected button html and add `m-3` for both buttons, so that there is consistent spacing regardless of whether they are on one line or two.

This CSS shouldn't be needed, since we rely on bootstrap to keep things consistent.